Snippets/mongodb/MongoDB_2/SharedTransactionDI.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace MongoDB_2
+{
+    using System.Threading.Tasks;
+    using NServiceBus.Storage.MongoDB;
+
+    public class SharedTransactionDI
```

Comment:
What's the value add of that outer class?

Reply:
namespacing the the types (e.g. `MyBusinessObject` which is also used in the other snippet)